cmov/README.md

```diff
@@ -0,0 +1,85 @@
+# [RustCrypto]: Conditional Move Intrinsics
+
+[![Crate][crate-image]][crate-link]
+[![Docs][docs-image]][docs-link]
+![Apache 2.0/MIT Licensed][license-image]
+![MSRV][rustc-image]
+[![Build Status][build-image]][build-link]
+
+Conditional move CPU intrinsics which are guaranteed to execute in
+constant-time and not be rewritten as branches by the compiler.
+
+Provides wrappers for the [CMOV family] of instructions on x86/x86_64 CPUs.
+
+[Documentation][docs-link]
+
+## About
+
+Conditional move intrinsics are a form of [predication] which allows selection
+of one or more values without using branch instructions, thus making the
+selection constant-time with respect to the values, and not subject to CPU
+execution features which might introduce timing or other microarchitectural
+sidechannels introduced by branch prediction or other speculative execution
+features.
+
+Intel has confirmed that all extant CPUs implement the CMOV family of
+instructions in constant-time, and that this property will hold for future
+Intel CPUs as well.
+
+This crate provides wrappers for the CMOV instructions implemented in terms
+of inline assembly as stabilized in Rust 1.59. This means the implementation
+is a black box that will not be rewritten by e.g. LLVM's architecture-specific
+lowerings, such as the [x86-cmov-conversion] pass.
+
+## Supported target architectures
+
+This crate will compile on any Rust target, however the intrinsics are only
+made available on specific target architectures listed below:
```

Comment:
Alternatively I could make it `compile_error!` on unsupported architectures. That might make more sense.

Reply:
Updated the crate to use the `compile_error!` approach for now